### ISSUE TITLE
Questions about transfer learning on COCO with DetCon pre-train

### ISSUE THREAD

**Issue by liming-ai** [deepmind/detcon#7]
Hi, @skoppula 

Thanks for your effort and contribution to self-supervised learning. Your recent paper Efﬁcient Visual Pretraining with Contrastive Detection (DetCon) inspired me a lot. However, there are still some issues bothering me, hope you can reply to me in your free time.

When transferring the pre-trained model to COCO, the fine-tuning recipe of DetCon is different with standard 1x schedule:

1. Learning rate is 0.3 for DetCon, the standard 1x is 0.2 for batch size 16
2. Weight decay is 4e-5 for DetCon, while 1e-4 for standard 1x
3. training and testing resolution is 1024 for DetCon while 1333x800 for standard 1x

I've noticed that other methods are almost always fine-tuned with the standard 1x, and I wonder why DetCon uses different hyperparameters, and if it's fair to do so.

If we need to adjust the parameters during fine-tuning for different pre-training models, do you have any tips and suggestions for parameter tuning?

Thanks again for your contribution and hope you can reply soon!

**Comment by olivierhenaff** [comment]
Hi Ming,

Thanks for your interest in our work! 

Indeed, we use the Cloud TPU Mask-RCNN implementation (https://github.com/tensorflow/tpu/tree/master/models/official/detection) when transferring to COCO, which differs from the detectron implementation in the ways you mention. However, we verified that our baselines (e.g. supervised and SimCLR pre-training) match those of published works (e.g. https://arxiv.org/pdf/2011.10043.pdf) so we don't think these implementation differences result in a significant change in performance.

All the best,

Olivier
